src/txdb-leveldb.cpp

```diff
@@ -619,32 +616,31 @@ bool CTxDB::LoadBlockIndex()
     nLoaded=pindex->nHeight;
     for ( ; pindex ; pindex= pindex->pnext )
     {
-
-        if( pindex->IsUserCPID() && pindex->cpid == uint128() )
+        if(IsResearchAgeEnabled(pindex->nHeight))
```

Comment:
This is to avoid repairing non-RA-blocks.

Reply:
Yes. No use doing irrelevant work...